packages/debs/SPECS/debian/rules

```diff
@@ -58,17 +58,17 @@ override_dh_install:
 
 	# Copying to target
 	mkdir -p $(TARGET_DIR)$(INSTALLATION_DIR)/tmp
-	mkdir -p $(TARGET_DIR)$(INSTALLATION_DIR)/bin
 	mkdir -p $(TARGET_DIR)$(INSTALLATION_DIR)/run
 	mkdir -p $(TARGET_DIR)$(INSTALLATION_DIR)/etc
 	mkdir -p $(TARGET_DIR)$(INSTALLATION_DIR)/var/lib
 	mkdir -p $(TARGET_DIR)$(INSTALLATION_DIR)/var/log
-	mkdir -p $(TARGET_DIR)$(INSTALLATION_DIR)/usr/share
+	mkdir -p $(TARGET_DIR)$(INSTALLATION_DIR)/usr/share/wazuh-server/bin
```

Comment:
I don't know if this change is needed. What happens if you leave like before?

Reply:
@RamosFe please, don't leave this comment unanswered and unresolved.

Reply:
This line is necessary because the `/usr/share/wazuh-server/bin` folder doesn't exists in the `$(TARGET_DIR)$(INSTALLATION_DIR)` folder, where the files are moved for the creation of the package.

When we delete this line, the following error occurs:

```console
# Copying to target
mkdir -p /build_wazuh/server/wazuh-server-5.0.0/debian/wazuh-server//tmp
mkdir -p /build_wazuh/server/wazuh-server-5.0.0/debian/wazuh-server//run
mkdir -p /build_wazuh/server/wazuh-server-5.0.0/debian/wazuh-server//etc
mkdir -p /build_wazuh/server/wazuh-server-5.0.0/debian/wazuh-server//var/lib
mkdir -p /build_wazuh/server/wazuh-server-5.0.0/debian/wazuh-server//var/log
cp -p /usr/share/wazuh-server/bin/wazuh-engine /build_wazuh/server/wazuh-server-5.0.0/debian/wazuh-server/usr/share/wazuh-server/bin/
cp: cannot create regular file '/build_wazuh/server/wazuh-server-5.0.0/debian/wazuh-server/usr/share/wazuh-server/bin/': No such file or directory
make[1]: *** [debian/rules:47: override_dh_install] Error 1
make[1]: Leaving directory '/build_wazuh/server/wazuh-server-5.0.0'
make: *** [debian/rules:30: binary] Error 2
dpkg-buildpackage: error: sudo debian/rules binary subprocess returned exit status 2
debuild: fatal error at line 1182:
dpkg-buildpackage -rsudo -us -uc -ui -b -nc failed
+ return 1
+ return 1
+ clean 1
+ exit_code=1
+ find /home/federamos/Documents/Wazuh/Repositories/wazuh/packages/debs/amd64 '(' -name '*.sh' -o -name '*.tar.gz' -o -name 'wazuh-*' ')' '!' -name docker_builder.sh -exec rm -rf '{}' +
+ exit 1
```